app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -71,8 +69,8 @@ class MainActivity : AppCompatActivity() {
                     val jsonObjectMain = jsonResponse.getJSONObject("main")
                     //Get the values of temp, pressure, and humidity from Json object main
                     //We subtract it to get temperature in celsius
-                    val temp = jsonObjectMain.getDouble("temp") - 273.15
-                    val feelsLike = jsonObjectMain.getDouble("feels_like") - 273.15
+                    val temp = jsonObjectMain.getDouble("temp") - TEMP_NUMBER
+                    val feelsLike = jsonObjectMain.getDouble("feels_like") - TEMP_NUMBER
```

Comment:
I created a constant val called TEMP_NUMBER which can be used instead of hard coding the value, I did this because it is something I have seen the professor do before and it is also better to do this instead of hardcoding a value.

Reply:
I genuinely appreciate your point of view and your attention to this detail. I agree with you, using a constant Val is better than hard coding the value. I will be implementing this in the code.

Reply:
Good idea, though I'd name it something descriptive.  TEMP_NUMBER sounds like a temporary variable name attached to a data type.